deployment/docker-swarm/database-utilities.sh

```diff
@@ -9,27 +9,43 @@ file=
 databaseKeyword=alertdb
 databaseName=alertdb
 userName=sa
+type="plain"
 
 # functions
 usage() {
   echo "usage: database-utilities - backup or restore a database with docker."
   echo
-  echo "database-utilities.sh [-b] [-d databaseName] [-f file] [-k containerKeyword] [-p] [-r] [-u userName]"
+  echo "database-utilities.sh [-b] [-d databaseName] [-f file] [-k containerKeyword] [-p] [-r] [-t type] [-u userName]"
   echo "Options: "
   echo "  -b: backup a database to the file specified in the file option."
   echo "  -d: the name of the database."
   echo "  -f: the file to save a backup or the file to restore the database from."
   echo "  -k: the keyword to search for the database container."
   echo "  -p: plain text database dump format"
   echo "  -r: restore a database from the file specified by the file option."
+  echo "  -t: the format for the backup or restore file of 'plain' or 'binary'."
   echo "  -u: database user name."
   echo "  -h: display this help."
   echo
   echo "Examples:"
+  echo " ------------------"
+  echo " Plain Text Format:"
+  echo " ------------------"
+  echo "  Backup (default):"
+  echo "    database-utilities.sh -n my-namespace -b -f ~/my-db-backup.dump"
+  echo "  Restore (default):"
+  echo "    database-utilities.sh -n my-namespace -r -f ~/my-db-backup.dump"
   echo "  Backup:"
-  echo "    database-utilities.sh -b -f ~/my-db-backup.dump"
+  echo "    database-utilities.sh -n my-namespace -b -t plain -f ~/my-db-backup.dump"
   echo "  Restore:"
-  echo "    database-utilities.sh -r -f ~/my-db-backup.dump"
+  echo "    database-utilities.sh -n my-namespace -r -t plain -f ~/my-db-backup.dump"
+  echo " ------------------"
+  echo " Binary Format:"
+  echo " ------------------"
+  echo "  Backup:"
+  echo "    database-utilities.sh -n my-namespace -b -t binary -f ~/my-db-backup.dump"
+  echo "  Restore:"
+  echo "    database-utilities.sh -n my-namespace -r -t binary -f ~/my-db-backup.dump"
```

Comment:
Probable cut-n-paste issue, but the docker-swarm script does not have the "-n" option.

Reply:
Help text updated.